pages.ko/common/ag.md

```diff
@@ -0,0 +1,33 @@
+# ag
+
+> The Silver Searcher.
+> ack과 비슷하지만, 더 빠르다. 
+> 더 많은 정보: <https://github.com/ggreer/the_silver_searcher>.
+
+- Find files containing "foo"를 포함하고 있는 파일들을 찾고, 내용에서 일치하는 행을 출력:
```

Comment:
Find files containing "foo" 

Is this phrase needed here?

Reply:
Oh, its my mistake. I'll modify that.